src/vlinder/visualize.py

```diff
@@ -212,18 +223,78 @@ def _create_table(self, key: str, **kwargs) -> pd.DataFrame.style:
         :param key: key of the values for the table
         :return: a styled table
         """
-        table_data = self._format_data_for_visual(key)
-
-        # Filter the data based on potentially provided arguments by the user.
-        table_data, name_str = self._apply_filters(table_data, **kwargs)
-        table_data = (
-            table_data.set_index(["scenario", key])
-            .pivot(columns="decision_makers_option", values="value")
-            .rename_axis((None, None))
-            .rename_axis(None, axis=1)
-        )
-        table_name = f"Values of {self._str_snake_case_to_text(key)}{name_str}"
-        return self._table_styler(table_data.style, table_name)
+        if key == "scenarios" or key == "fixed_inputs" or key == "decision_makers_options" \
```

Comment:
Is there a way to simply this? We can also have a look at this together next week if you want

Reply:
Fixed to 'key in []'